tests/components/zha/test_sensor.py

```diff
@@ -350,6 +356,14 @@ async def async_test_powerconfiguration(hass, cluster, entity_id):
             },
             None,
         ),
+        (
+            general.DeviceTemperature.cluster_id,
+            "device_temperature",
+            async_test_device_temperature,
+            1,
```

Comment:
I have no idea if value `1` is correct. `0` was failing, so I wrote `1`.

Reply:
It's the number of attribute reports the tests expects and entity shouldd receive. 1 is correct for your test